### ISSUE TITLE
Only able to divide stake when staking address is explicitly provided

### ISSUE THREAD

**Issue by derekpierre** [nucypher/nucypher#2113]
**Describe the Bug**
I encountered an interesting bug when using `nucypher stake divide`. When running the command without specifying my staking address I get the following:
```
$ nucypher stake divide
	Account
--  ------------------------------------------
 0  0x63e478bc474eBb6c31568ff131cCd95C24bfD552
 1  0x270b3f8af5ba2B79ea3Bd6a6Efc7ecAB056d3E3f
 2  0x45D33d1Ff0A7E696556f36DE697E5C92C2CCcFaE
Select index of staking account [0]: 1
Selected 1: 0x270b3f8af5ba2B79ea3Bd6a6Efc7ecAB056d3E3f
NOTE: Showing divisible stakes only
No divisible stakes found.
```

However if I specify the same staking address explicitly as a parameter then I'm allowed to divide the stake:
```
(nucypher) derek@derek-home-server:~/nucypher/nucypher$ nucypher stake divide --staking-address 0x270b3f8af5ba2B79ea3Bd6a6Efc7ecAB056d3E3f
NOTE: Showing divisible stakes only

Network Ibex ═════════════════════════════════════════
Staker 0x270b3f8af5ba2B79ea3Bd6a6Efc7ecAB056d3E3f ════
Worker 0x45D33d1Ff0A7E696556f36DE697E5C92C2CCcFaE ════
--------------  ----------------
Status      	Committed #18436
Restaking   	Yes (Unlocked)
Winding Down	No
Unclaimed Fees  0 ETH
Min fee rate	0 ETH
--------------  ----------------
╒═══════╤═════════════════════════════╤═════════════╤═════════════╤═══════════════╕
│   Idx │ Value                   	│   Remaining │ Enactment   │ Termination   │
╞═══════╪═════════════════════════════╪═════════════╪═════════════╪═══════════════╡
│ 	0 │ 31437.841006996376688377 NU │      	31 │ Jun 19 2020 │ Jul 22 2020   │
╘═══════╧═════════════════════════════╧═════════════╧═════════════╧═══════════════╛
Select Stake: 0
Enter target value (15000 NU - 16437.841006996376688377 NU): 15000
Enter number of periods to extend: 20

══════════════════════════════ ORIGINAL STAKE ════════════════════════════

Staking address: 0x270b3f8af5ba2B79ea3Bd6a6Efc7ecAB056d3E3f
~ Original Stake: | - | 0x270b | 0x45D3 | 0 | 31437.841006996376688377 NU | 33 periods . | Jun 19 20:00 EDT - Jul 22 20:00 EDT


══════════════════════════════ STAGED STAKE ══════════════════════════════

Staking address: 0x270b3f8af5ba2B79ea3Bd6a6Efc7ecAB056d3E3f
~ Chain  	-> ID # 4 | Rinkeby
~ Value  	-> 15000 NU (15000000000000000000000 NuNits)
~ Duration   -> 53 Days (53 Periods)
~ Enactment  -> Jun 19 20:00 EDT (period #18433)
~ Expiration -> Aug 11 20:00 EDT (period #18486)
    
═════════════════════════════════════════════════════════════════════════
Publish stake division to the blockchain? [y/N]: y
Enter password to unlock account 0x270b3f8af5ba2B79ea3Bd6a6Efc7ecAB056d3E3f:
Confirm transaction DIVIDESTAKE on hardware wallet... (76058 gwei @ 1000000000)
Broadcasting DIVIDESTAKE Transaction (76058 gwei @ 1000000000)...
Successfully divided stake
OK | 0x74ddd647de6eaca7ef0c485706ef526001d959a3c2eaa98699e087a7d259d08b (75349 gas)
Block #6711982 | 0xd1c6d6df257ecd05632550565edb709ae577066a60ca433bc4d23de5fb332009
 See https://rinkeby.etherscan.io/tx/0x74ddd647de6eaca7ef0c485706ef526001d959a3c2eaa98699e087a7d259d08b
...
```

My stakeholder configuration:
```
(nucypher) derek@derek-home-server:~/nucypher/nucypher$ nucypher stake config
Stakeholder Configuration /home/derek/.local/share/nucypher/stakeholder.json
 =======================================================
{
  "provider_uri": “<infura>",
  "poa": null,
  "light": null,
  "domains": [
    "ibex"
  ],
  "checksum_addresses": [],
  "signer_uri": "clef:///home/derek/.clef/clef.ipc",
  "version": 1
}
```

Also, there is a bug in the string parameter we pass to our `PROMPT_STAKE_DIVIDE_VALUE` template string:
```
  File "/home/derek/nucypher/nucypher/nucypher/cli/commands/stake.py", line 664, in divide
	prompt = PROMPT_STAKE_DIVIDE_VALUE.format(minimm=min_allowed_locked, maximum=str(max_divide_value))
KeyError: 'minimum'
```
i.e. `minimm` should be `minimum`. I manually changed it locally to proceed.

**To Reproduce**
See above.

**Traceback or Screenshots (Optional) **
See above

**System (please complete the following information):**
 - OS Version: Linux Ubuntu 18.04.4
 - Nucypher Version: `master` @ https://github.com/nucypher/nucypher/commit/202e502ad4239227b613c1346cd161723d892765
 - Ethereum Node Version:  Geth 1.9.15-stable-0f77f34b

**Comment by vzotova** [comment]
related #2116

**Comment by derekpierre** [comment]
Problem with `PROMPT_STAKE_DIVIDE_VALUE` template string is fixed in #2104
